exporter/stats/stackdriver/stackdriver.go

```diff
@@ -119,7 +117,23 @@ func (e *Exporter) Export(vd *stats.ViewData) {
 	if len(vd.Rows) == 0 {
 		return
 	}
-	e.bundler.Add(vd, 1)
+	err := e.bundler.Add(vd, 1)
+	switch err {
+	case nil:
+		return
+	case bundler.ErrOversizedItem, bundler.ErrOverflow:
+		go e.handleUpload(vd)
```

Comment:
It might be a bit aggressive to do this on ErrOverflow. In the case of lack of resources, we might not be consuming the bundler fast enough. By creating more coroutines, we will contribute to the problem.

Reply:
Hmm, interesting. Would you recommend instead that we pause for a couple of seconds in the case of ErrOverflow, or should we just drop it?

Reply:
Dropping and logging it sounds reasonable.